package.json

```diff
@@ -14,90 +14,41 @@
     "deploy-docs": "npm run generate-docs & npm run build-docs & git subtree push --prefix docs/dist origin gh-pages",
     "dev-docs": "npm run generate-docs & webpack-dev-server --port 3000 --config docs/webpack.config.dev.js --content-base docs",
     "generate-docs": "node_modules/react-docgen/bin/react-docgen.js src/components/victory-line.jsx -o docs/victory-line.json",
     "hot-docs": "npm run generate-docs & webpack-dev-server --port 3000 --config docs/webpack.config.hot.js --hot --content-base docs",
-    "open-demo": "opener http://127.0.0.1:3000",
-    "open-dev": "npm run dev & npm run open-demo",
-    "open-hot": "npm run hot & npm run open-demo",
-    "lint-node": "eslint -c .eslintrc-node *.js demo/webpack.*.js",
-    "lint-react": "eslint --ext .js,.jsx -c .eslintrc-react src demo/*.jsx",
-    "lint-react-test": "eslint --ext .js,.jsx -c .eslintrc-react-test src test/client",
-    "lint": "npm run lint-node && npm run lint-react && npm run lint-react-test",
-    "test-frontend": "node node_modules/karma/bin/karma start karma.conf.js",
-    "test-frontend-ci": "node node_modules/karma/bin/karma start --browsers PhantomJS,Firefox karma.conf.coverage.js",
-    "test-frontend-cov": "node node_modules/karma/bin/karma start karma.conf.coverage.js",
-    "test-frontend-dev": "node node_modules/karma/bin/karma start karma.conf.dev.js",
-    "test": "npm run test-frontend && echo 'TODO Server 13, Integration 12 tests.'",
-    "test-ci": "npm run test-frontend-ci",
-    "test-cov": "npm run test-frontend-cov",
-    "test-dev": "npm run test-frontend-dev",
-    "check": "npm run lint && npm run test",
-    "check-ci": "npm run lint && npm run test-ci",
-    "check-cov": "npm run lint && npm run test-cov",
-    "check-dev": "npm run lint && npm run test-dev",
     "build-static": "webpack --config docs/webpack.config.static.js --progress",
     "push-gh-pages": "git subtree push --prefix gh-pages origin gh-pages"
   },
   "dependencies": {
-    "babel": "^5.5.8",
-    "babel-core": "^5.5.8",
-    "babel-loader": "^5.3.2",
+    "builder": "2.1.0",
+    "builder-react-component": "0.1.0",
     "d3": "^3.5.6",
     "lodash": "^3.9.3",
     "radium": "^0.14.1",
-    "rimraf": "^2.4.0",
     "style-loader": "~0.8.0",
-    "url-loader": "~0.5.5",
-    "victory-animation": "^0.0.11",
+    "victory-animation": "^0.0.10",
     "victory-label": "0.1.1",
-    "webpack": "^1.10.0"
+    "victory-util": "2.0.0"
   },
   "devDependencies": {
-    "babel-eslint": "^3.1.25",
+    "babel": "^5.5.8",
+    "babel-core": "^5.5.8",
+    "babel-loader": "^5.3.2",
+    "builder-react-component-dev": "0.1.0",
     "chai": "^3.2.0",
     "clean-webpack-plugin": "^0.1.4",
     "ecology": "git+https://github.com/formidablelabs/ecology#master",
-    "eslint": "^0.24.1",
-    "eslint-config-defaults": "^3.0.3",
-    "eslint-plugin-filenames": "^0.1.1",
-    "eslint-plugin-react": "^2.6.4",
     "file-loader": "^0.8.4",
     "isparta-loader": "^0.2.0",
```

Comment:
I think some of these `devDependencies` we can ditch (I know at least `isparta-loader` isn't used by the docs webpack), but we can clean up the remaining deps when we get a victory archetype – no big deal for now.